src/services.js

```diff
@@ -1,29 +1,104 @@
-'use strict'
-const async = require('async')
-const cp = require('child_process')
-const formatOutput = require('./format-output')
-const services = [
+const async = require('async');
+const cp = require('child_process');
+const formatOutput = require('./format-output');
+const platformHelper = require('./utils/platform-helper.js');
+
+/**
+ * Service names.
+ * @private
+ *
+ * @const {string[]}
+ */
+const SERVICES = [
   'apache2',
+  'nginx',
   'steelpenny',
   'webpack-dev-server',
-  'nginx'
-]
-
-const me = {}
-
-me.bulkAction = function (opts, cb) {
-  const action = opts.action
-  async.map(
-    services,
-    (service, cb) => {
-      let cmd = `sudo service ${service} ${action}`
-      cp.exec(cmd, (err, stderr, stdout) => {
-        if (err) return cb(null, { service, error: err.message })
-        return cb(null, { service, output: formatOutput(opts.verbose, stderr, stdout) })
-      })
-    },
-    cb
-  )
+];
+
+/**
+ * Get an OS-appropriate command for a service.
+ * @private
+ *
+ * @param {string} service
+ * @param {string} action
+ * @param {getServiceCommandCallback} cb Callback
+ */
+function getServiceCommand(service, action, cb) {
+  platformHelper.getIsUbuntu16((error, isUbuntu16) => {
```

Comment:
Placed the “am I Ubuntu 16?” functionality in a separate module (_src/utils/platform-helper.js_) to make testing possible.